.github/maintainers_guide.md

```diff
@@ -81,14 +83,20 @@ Releasing can feel intimidating at first, but rest assured: if you make a mistak
 
     - Close the relevant GitHub Milestone(s) for the release(s)
 
+    - Create a new GitHub Milestone for new issues to live. Typically, you'll create a new minor version - however, if there are any bugs that need to be carried over from the current GitHub Milestone, you could make a Milestone for a patch version to reflect those issues
```

Comment:
So I think we could use some tweaks here, actually. After we finished our huddle, I ran through the same steps for the `@slack/oauth` package. In that situation, I was releasing a patch version (2.5.4), and the next minor version milestone (2.6.0) already existed.

With that context described, perhaps we could change this line to suggest to first look and see if the next minor version milestone is already created before creating a new minor version milestone? Something like that? Or "If the next minor version milestone does not exist, create it" or something along those lines? What do you think? Kind of tricky to communicate this in a succinct bullet point lol. It doesn't have to be perfect and we can always improve on it later.